### PR TITLE
Add detail page

### PR DESCRIPTION
**Pull request by khizarkhan345**

### CODE REVIEW

**Comment by khizarkhan345** [comment]
The second commit was automatically done due to a merge conflict with the main branch. That's why we have that commit message. I don't know how to change that commit message.

**Comment by Lawangin** [comment]
I added a function that utilizes the `\n` in the description and breaks a new line. It looks complex, but it's to adjust for the occasional "\n\n\n\n\n" that I see.

File: src/app/components/DataDetailPage.tsx [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+import { Fragment } from 'react';
+import { Box, Flex, Text, Icon, Link, Image } from '@chakra-ui/react';
+import { MdLocationOn } from 'react-icons/md';
+import { RiBuildingFill } from 'react-icons/ri';
+import { AiOutlineDollar, AiOutlineClose } from 'react-icons/ai';
+import { TbBrandGoogleAnalytics } from 'react-icons/tb';
+import styled from 'styled-components';
+
+const Btn = styled.button`
```
Comment (Lawangin) [inline src/app/components/DataDetailPage.tsx:11]: Need to gray border to the pop-up page from the designs.

image

Comment (Lawangin) [reply src/app/components/DataDetailPage.tsx:11]: should look like this:
image

File: src/app/components/DisplayData.tsx [hunk 2]
```diff
@@ -14,22 +16,40 @@ const Btn = styled.button`
...
+  Function,
+  Location,
+  Industry,
   Level,
   URL,
+  image_url,
 }: ChildComponentProps) {
+  const [popUpState, setPopState] = useState<boolean>(false);
+
+  const handlePopUpState = () => {
```
Comment (Lawangin) [inline src/app/components/DisplayData.tsx:49]: background should be grayed out and when clicked, it should close the pop-up.

File: src/app/components/DataDetailPage.tsx [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+  Level: string;
+  Type: string;
+  Function: string;
+  Industry: string;
+  URL: string;
+  image_url: string;
+  handlePopUpState: () => void;
+};
+
+export default function DataDetailPage({
```
Comment (Lawangin) [inline src/app/components/DataDetailPage.tsx:41]: The text and components should be centered among the pop space and the space given to them should be wider.
image

Comment (Lawangin) [reply src/app/components/DataDetailPage.tsx:41]: In addition, The `Entry Level` with the icon should start at 50% of the content or just split to the ends if that is easier.

**Review by Lawangin — CHANGES_REQUESTED** [review]

File: src/app/components/DataDetailPage.tsx (hunk 3 above)
Comment (khizarkhan345) [reply src/app/components/DataDetailPage.tsx:42]: I am using an icon and have not found a way to fill half of it.

**Comment by khizarkhan345** [comment]
The build has failed because there is eslint error in my code which is due to the function splitLines written by you. The error says that splitLines is defined but it's never used.

**Review by Lawangin — APPROVED** [review]

**Pull request merged**